index.html

```diff
@@ -0,0 +1,12 @@
+<head>
+    <link rel="stylesheet" href="styles.css">
+     </head>
+   <main id="main">
+     <title id="title">DR NORMAN BORLAUG</title>
+     <div id="img-div">
+       <img id="image"></img>
+       <figcaption id="img-caption">This is about Dr. Norman Borlaug</figcaption>
+        </div>
+        <article id="tribute-info">This is Dr. Norman Borlaug who helped alot of people </article>
+        <a href="#" target="_blank" id="tribute-link">About</a>
+   </main>
```

Comment:
Hello Mark, thank you for the efforts so far but there are a few things that you really need to change and include in your project so as to be approved.
Read the comments below and make the appropriate changes.

1. You need to use sematic tags in your code because your code lacks the HTML and body tags.
2. Your code should have a proper HTML5 boiler plate i.e <!DOCTYPE html> that alerts the browser that it is dealing with an HTML5 document.